### PR TITLE
Add optional "business_unit" to customer invoice

### PR DESCRIPTION
**Pull request by kaleposhobios**

### CODE REVIEW

File: brokers/rest.md [hunk 1]
```diff
@@ -957,20 +957,24 @@ Request:
...
+    "shipments": [                                     // Required
       {"external_id": "123"},
       {"external_id": "456"}
     ],
-    "loads": [                                     // Optional
+    "loads": [                                         // Optional
       {"external_id": "123"}
-    ]
+    ],
+    "billing_unit": {                                  // Optional, if this invoice belongs to a specific billing unit inside your account.
```
Comment (r38y) [inline brokers/rest.md:974]: Billing unit or business unit?

File: brokers/rest.md [hunk 2]
```diff
@@ -957,20 +957,24 @@ Request:
...
       {"external_id": "456"}
     ],
-    "loads": [                                     // Optional
+    "loads": [                                         // Optional
       {"external_id": "123"}
-    ]
+    ],
+    "business_unit": {                                 // Optional, if this invoice belongs to a specific billing unit inside your account.
+      "external_id": "business-unit-external-id",      // Required
+      "name": "example-name"                           // Optional, recommended if new billing unit
```
Comment (r38y) [inline brokers/rest.md:976]: I'd make name required here and in the model since just external_id doesn't really do anything for us.

**Review by r38y — CHANGES_REQUESTED** [review]

File: brokers/rest.md (hunk 2 above)
Comment (kaleposhobios) [reply brokers/rest.md]: Thanks!

**Review by r38y — APPROVED** [review]

**Review by r38y — APPROVED** [review]

**Review by kaleposhobios — COMMENTED** [review]
Ok!

**Pull request merged**